packages/orbit-components/src/Wizard/__examples__/DEFAULT.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import * as React from "react";
+
+import Box from "../../Box";
+import Button from "../../Button";
+import Heading from "../../Heading";
+import Stack from "../../Stack";
+import Text from "../../Text";
+import Wizard from "../index";
+import WizardStep from "../WizardStep";
+
+export default {
+  Example: () => {
+    const [activeStep, setActiveStep] = React.useState(2);
+    const [headingAndText, setHeadingAndText] = React.useState({
+      heading: "Ticket fare",
+      text: "Select your ticket fare.",
+    });
+    const handleStepChange = step => {
+      setActiveStep(step);
+      switch (step) {
+        case 0:
+          setHeadingAndText({
+            heading: "Search",
+            text: "Find a connection.",
+          });
+          break;
+        case 1:
+          setHeadingAndText({
+            heading: "Passenger details",
+            text: "Enter details for all passengers.",
+          });
+          break;
+        case 2:
+          setHeadingAndText({
+            heading: "Ticket fare",
+            text: "Select your ticket fare.",
+          });
+          break;
+        case 3:
+          setHeadingAndText({
+            heading: "Customize your trip",
+            text: "Select seating.",
+          });
+          break;
+        default:
+          setHeadingAndText({
+            heading: "Overview and payment",
+            text: "Enter payment details.",
+          });
+      }
+    };
```

Comment:
i would limit number of clicks, it should not go to infinity :) 

now if im clicking on `next step` it goes beyond number of steps, it should return to the first step i think

Reply:
😱 ♾️

Reply:
Infinity is bad in this case. Because it's supposed to show default behavior, I'd go for hiding the button on the last step rather than cycling back (so it doesn't seem like that's the default way to get to the start).